components/raftstore/src/store/fsm/apply.rs

```diff
@@ -941,7 +959,7 @@ impl ApplyDelegate {
             );
         }
 
-        if cmd.has_admin_request() {
+        if should_write_to_engine_after(&cmd) {
```

Comment:
You can write `apply_ctx.syn_log_hint |= xxx`.

Reply:
Addressed